src/pages/createConcert/CreateConcert.js

```diff
@@ -5,26 +5,76 @@ import { Button, Typography } from '@mui/material';
 import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
 import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
 import { DateTimePicker } from '@mui/x-date-pickers/DateTimePicker';
+import { collection, addDoc } from 'firebase/firestore';
+import { db } from '../../configs/firebase';
 
 const CreateConcert = () => {
-  //Concert form state
+  //Concert form states
   const [title, setTitle] = useState('');
   const [venue, setVenue] = useState('');
-  const [date, setDate] = useState(Date);
-  const [image, setImage] = useState('');
+  const [time, setTime] = useState(null);
+  console.log(time);
+  const [coverImage, setCoverImage] = useState('');
+  const [artists, setArtists] = useState(['Nishant', 'Prashant', 'Hriday']);
+  const [titleError, setTitleError] = useState(false);
+  const [venueError, setVenueError] = useState(false);
+  const [dateError, setDateError] = useState(false);
+  const [imageError, setImageError] = useState(false);
 
   // Handle form submission
-  const handleSubmit = (e) => {
+  const handleSubmit = async (e) => {
```

Comment:
In useFirestore hook make addDocument function and call it from there by passing required params. Which will be more reusable. And also manage data, error, isPending states there only.